dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -69,6 +69,10 @@ public enum ErrorCode {
 
   E1122("Category option combo {0} cannot be associated with the default category combo"),
   E1125("Category option combo {0} contains options not associated with category combo {1}"),
+  E1126("Category combo {0} cannot combine more than {1} categories, but had: {2}"),
+  E1127("Category {0} cannot have more than {1} options, but had: {2} "),
+  E1128("Category combo {0} cannot have more than {1} combinations, bud requires: {2}"),
```

Comment:
maybe this message can mirror the other 2?
`..... cannot have more than x combinations, but had: y"

Reply:
It is worded differently because it is different. The combinations do not exist (yet) and when the validation fails we also don't create them, so I thought wording it "but had" is confusing if you know what is going on.

Reply:
ok cool. There is a typo in `bud`. Should be `but`.